<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotItem</class>
 <widget class="QWidget" name="BotItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>96</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>96</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>96</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hl_control">
     <item>
      <widget class="QLineEdit" name="le_username">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_password">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_proxy">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_type">
       <item>
        <property name="text">
         <string>2:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7:30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Random</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_edit">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_testProxy">
       <property name="text">
        <string>Test Proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_stop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="w_bot" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="l_points">
        <property name="text">
         <string>Piracy Points: 0    Crew Points: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>192</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_show">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_hide">
        <property name="text">
         <string>Hide</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_attack">
        <property name="text">
         <string>Attack</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_train">
        <property name="text">
         <string>Train</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_collect">
        <property name="text">
         <string>Collect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_pause">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_resume">
        <property name="text">
         <string>Resume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_status">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
